test/vtcomposite-localize-language.test.js

```diff
@@ -861,3 +860,157 @@ test('[localize language] _mbx_name_local exists in the input tile', (assert) =>
+    name_de: 'place DE',
+    _mbx_name_it: 'place', // same as `name`, expected to be dropped
+    _mbx_name_fr: 'place FR',
+  };
+  const expectedLocalizedProperties = {
+    name: 'place',
+    name_local: 'place',
+    name_de: 'place DE',
+    name_fr: 'place FR'
+  };
+  const params = {
+    buffer: mvtFixtures.create({
+      layers: [
+        {
+          version: 2,
+          name: 'places',
+          features: [
+            {
+              id: 10,
+              tags: Object.keys(rawDataLanguageProperties).flatMap((_, index) => [index, index]),
+              type: 1, // point
+              geometry: [9, 54, 38]
+            }
+          ],
+          keys: Object.keys(rawDataLanguageProperties),
+          values: Object.values(rawDataLanguageProperties).map((value) => ({ string_value: value })),
+          extent: 4096
+        }
+      ]
+    }).buffer,
+    languages: ['all']
+  };
+
+  localize(params, (err, buffer) => {
+    assert.notOk(err);
+    const info = vtinfo(buffer);
+    const properties = info.layers.places.feature(0).properties;
+    assert.equal(info.layers.places.length, 1, 'expected number of features');
+    assert.deepEqual(properties, expectedLocalizedProperties, 'expected properties');
+    assert.end();
+  });
+});
+
+test('[localize language] languages=all returns the last value of the same language properties', (assert) => {
+  const rawDataLanguageProperties = {
+    name: '你好',
+    name_script: 'Han',
+    'name_zh-Hant': 'Nǐ hǎo',
+    name_en: 'EN first value - to be overwritten',
+    _mbx_name_de: 'DE first value - to be overwritten',
+    name_de: 'hallo',
+    _mbx_name_it: 'ciao',
+    _mbx_name_fr: 'bonjour',
+    _mbx_name_en: 'hello'
+  };
+  const expectedLocalizedProperties = {
+    name: '你好',
+    name_local: '你好',
+    'name_zh-Hant': 'Nǐ hǎo',
+    name_en: 'hello',
+    name_de: 'hallo',
+    name_it: 'ciao',
+    name_fr: 'bonjour'
+  };
+  const params = {
+    buffer: mvtFixtures.create({
+      layers: [
+        {
+          version: 2,
+          name: 'places',
+          features: [
+            {
+              id: 10,
+              tags: Object.keys(rawDataLanguageProperties).flatMap((_, index) => [index, index]),
+              type: 1, // point
+              geometry: [9, 54, 38]
+            }
+          ],
+          keys: Object.keys(rawDataLanguageProperties),
+          values: Object.values(rawDataLanguageProperties).map((value) => ({ string_value: value })),
+          extent: 4096
+        }
+      ]
+    }).buffer,
+    languages: ['all']
+  };
+
+  localize(params, (err, buffer) => {
+    assert.notOk(err);
+    const info = vtinfo(buffer);
+    const properties = info.layers.places.feature(0).properties;
+    assert.equal(info.layers.places.length, 1, 'expected number of features');
+    assert.deepEqual(properties, expectedLocalizedProperties, 'expected properties');
+    assert.end();
+  });
+});
```

Comment:
A test with e.g. `languages: ['all', 'de', 'local', 'fi']` could add coverage for this [if-else case](https://github.com/mapbox/vtcomposite/pull/136/files#diff-effefce41d1b570607acee45c42ebdeabe9990a648d90164f333e03fd9f70cc8R690). Does that make sense?